ext/enchant/tests/dict_remove_from_session.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
+enchant_dict_remove_from_session() function
+--EXTENSIONS--
+enchant
+--SKIPIF--
+<?php
+if (!is_array(enchant_broker_list_dicts(enchant_broker_init()))) die("skip no dictionary installed on this machine!");
+?>
+--FILE--
+<?php
+$broker = enchant_broker_init();
+$dicts = enchant_broker_list_dicts($broker);
+$newWord = 'myImaginaryWord';
+
+$requestDict = enchant_broker_request_dict($broker, $dicts[0]['lang_tag']);
```

Comment:
Mind adding a test which includes a null byte in the string?

Reply:
Lovely, I wonder how many bugs could be found by fuzzing extension functions with strings that contain null bytes.

@YuanchengJiang you might want to try this in your fuzzer.

Reply:
I rebased this and added a commit to properly handle the null bytes.
Will squash ofc on merge.